client/views/chan.tpl

```diff
@@ -1,7 +1,15 @@
 {{#each channels}}
 <div data-id="{{id}}" data-target="#chan-{{id}}" data-title="{{name}}" class="chan {{type}} chan-{{slugify name}}">
 	<span class="badge{{#if highlight}} highlight{{/if}}">{{#if unread}}{{roundBadgeNumber unread}}{{/if}}</span>
+	{{#equal type "lobby"}}
+		<span class="tooltipped tooltipped-w tooltipped-no-touch" aria-label="Join channel...">
+			<button class="add-channel" aria-label="Add channel" data-id="{{id}}"></button>
```

Comment:
Same here in the ARIA label.

Reply:
I don't think the button needs aria label since the span above has one.